File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Municipio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Municipio
    {
        [Key]
        public int idMunicipio { get; set; }

        [MaxLength(20)]
        public string nombreMunicipio { get; set; }

        public int idDepartamento { get; set; }
        [ForeignKey("idDepartamento")]
        public Departamento Departamento { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Municipio.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Municipio
    <Key>
    Public Property idMunicipio As Integer
    <MaxLength(20)>
    Public Property nombreMunicipio As String
    Public Property idDepartamento As Integer
    <ForeignKey("idDepartamento")>
    Public Property Departamento As Departamento
End Class
